nvflare/private/fed/server/fed_server.py

```diff
@@ -327,6 +327,11 @@ def _register_cellnet_cbs(self):
             topic=CellChannelTopic.HEART_BEAT,
             cb=self.client_heartbeat,
         )
+        self.cell.register_request_cb(
```

Comment:
No need here.
Should just register a CB in server_runner.

Reply:
changed to use aux_message sending.